### ISSUE TITLE
Value "monthly" is not accepted as recurrency type

### ISSUE THREAD

**Issue by nattomi** [wichtounet/budgetwarrior#34]
Hi!

My problem is that the seemingly correct recurrence value `monthly` is not accepted. Consider the following session:

```
~$ budget account
Accounts 

ID Name    Amount     Part   
1  General 2000000.00 100,00%
   Total   2000000.00        

~$ budget recurring add
Account []: General
Name []: Test
Amount [0.00]: 100
Recurrence []: monthly
This value can only be one of these values [monthly, weekly]
Recurrence [monthly]:
```

I've compiled budgetwarrior (revision 15712d0) from source on Ubuntu 23.10. Is there anything I can do for debugging it further?

**Comment by wichtounet** [comment]
Hi 

Thanks for reporting that! I have fixed this in 395c8741790446d1b94593cb67cba383c357fe78
It was indeed a bug in the code 👍

**Comment by nattomi** [comment]
Thanks a lot! I can confirm that now it works as expected.
